### ISSUE TITLE
Error while loading documentation for Swift argument parser

### ISSUE THREAD

**Issue by heckj** [SwiftPackageIndex/SwiftPackageIndex-Server#3546]
When I tried to load the documentation hosted at SPI for swift-argument-parser, the site responded with "An unknown error occurred."

1. Navigate to https://swiftpackageindex.com/apple/swift-argument-parser/1.5.0/
2. See the error 

**Screenshots**

![IMG_0327](https://github.com/user-attachments/assets/286d68ac-fa7f-4572-b070-b0ee9071e008)

**What web browser were you using? (if relevant)**

If you're reporting a bug with the front end, please include your browser information below:

- Platform: iPad
- Browser Safari
- Version iOS 18.1.1

**Comment by finestructure** [comment]
Just to clarify - Joe's issue isn't a 404, at least not straight up. It might be for one of the core assets, which could then render this failure page in the Vue app, I suppose.

Did the original issue also show this behaviour at times or was it always a 404 page?

**Comment by daveverwer** [comment]
> Did the original issue also show this behaviour at times or was it always a 404 page?

Yes, this error is when it's a 404 or other error when one of the dynamically loaded files fails to load with an error.

**Comment by heckj** [comment]
It's been several hours since I first this this, but I just ran the commands:

```
(base) ➜  ~ curl -I https://swiftpackageindex.com/apple/swift-argument-parser/1.5.0/
HTTP/2 404
date: Tue, 10 Dec 2024 15:45:16 GMT
content-type: text/html; charset=utf-8
content-length: 5182
cache-control: no-store
cf-cache-status: EXPIRED
server: cloudflare
cf-ray: 8efe5c6fbb1cc39c-SEA

(base) ➜  ~ curl -I https://swiftpackageindex.com/apple/swift-argument-parser/1.5.0/
HTTP/2 404
date: Tue, 10 Dec 2024 15:45:20 GMT
content-type: text/html; charset=utf-8
content-length: 5182
cache-control: no-store
cf-cache-status: HIT
age: 4
server: cloudflare
cf-ray: 8efe5c857e41dedc-SEA
```

The error message shown at the page (https://swiftpackageindex.com/apple/swift-argument-parser/1.5.0/) has now changed  as well - on my Mac, it's now showing a more classic 404 response.

<img width="1174" alt="Screenshot 2024-12-10 at 7 46 41 AM" src="https://github.com/user-attachments/assets/06df354d-ce51-4f54-a7b5-f76ad81e5522">

**Comment by heckj** [comment]
I don't know what odd CF caching issue was bring troublesome, but I just iterated back through this today and this specific instance appears to be resolved. Closing.

**Comment by finestructure** [comment]
This seems to be affecting users in the UK fetching this URL right now: https://swiftpackageindex.com/grpc/grpc-swift/2.0.0-rc.1/js/866.7ae93681.js
